hs_access_control/models.py

```diff
@@ -1273,62 +1274,97 @@ def get_resources_with_explicit_access(self, this_privilege, via_user=True, via_
 
             if via_user and via_group:
                 query = uquery | gquery
-                excl = uexcl | gexcl
+                excl = uexcl | gexcl  # user privilege can squash group privilege
             elif via_user:
                 query = uquery
-                excl = uexcl
+                excl = None  # users have one privilege
             elif via_group:
                 query = gquery
-                excl = gexcl
+                excl = None  # groups cannot OWN
             else:
                 # nothing matches
-                return BaseResource.objects.empty()
+                return BaseResource.objects.none()
 
-            return BaseResource.objects\
-                .filter(query)\
-                .exclude(pk__in=BaseResource.objects
-                         .filter(excl)).distinct()
+            if excl:
+                return BaseResource.objects\
+                    .filter(query)\
+                    .exclude(pk__in=BaseResource.objects
+                             .filter(excl)).distinct()
+            else:
+                return BaseResource.objects\
+                    .filter(query).distinct()
 
-        else:  # this_privilege == PrivilegeCodes.ViEW
+        else:  # this_privilege == PrivilegeCodes.VIEW
             # VIEW includes CHANGE+immutable as well as explicit VIEW
             # TODO: make this query more efficient, if possible!
             # CHANGE does not include immutable resources
+
+            # step one: compute view resources explicitly:
             uquery = Q(r2urp__privilege=this_privilege,
                        r2urp__user=self.user)
             uexcl = Q(r2urp__privilege__lt=this_privilege,
```

Comment:
See line 1314. This is the line that allows user privilege to override group privilege when both are looked at together. For example, suppose someone has VIEW from a group, and OWNER as a user. This line excludes owned resources from the VIEW resources.